### PR TITLE
Home Sections, Model and Interfaces - PT:1

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#12532

*Description of changes:*
*NOTE:* Initial commit to avoid code conflicts as there is multiple devs working on the same steps, there will be a follow up PR for completion.

* Updates Step 1 architecture
* Implements AddEdit Home Section Form
* Implements Baseline Table to display home sections with dummy data
* Implements Mhr Registrations Interfaces and store intialization
* Updated directories
* Fixes Footer Help content which identifies App version numbers

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/interfaces/mhr-registration-interfaces/HomeSectionIF.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// New mhr registration Home Section interface.
+export interface HomeSectionIF {
```
Comment (doug-lovett) [inline ppr-ui/src/interfaces/mhr-registration-interfaces/HomeSectionIF.ts:2]: Just an observation, I think inches will have a default value of 0.

File: ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import { HomeSectionIF } from '@/interfaces'
+
+export interface MhrRegistrationIF {
+  clientReferenceId: string,
+    declaredValue: string,
+    registeringParty: {
```
Comment (doug-lovett) [inline ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts]: In the API registeringParty will be renamed to submittingParty and will have phoneNumber and phoneExtension properties.

File: ppr-ui/src/store/state/state-model.ts [hunk 3]
```diff
@@ -88,7 +88,76 @@ export const stateModel: StateModelIF = {
       debtors: []
     }
   },
-  mhrRegistration: {},
+  mhrRegistration: {
+    clientReferenceId: '',
+    declaredValue: '',
+    registeringParty: {
```
Comment (doug-lovett) [inline ppr-ui/src/store/state/state-model.ts]: Same as previous comment.

**Review by doug-lovett — APPROVED** [review]
Looks great, just a couple of observations.

File: ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts]: Awesome, i recall this being mentioned now. 
Will update, thank you!

File: ppr-ui/src/store/state/state-model.ts (hunk 3 above)
Comment (cameron-eyds) [reply ppr-ui/src/store/state/state-model.ts]: Thank you, will update!

File: ppr-ui/src/interfaces/mhr-registration-interfaces/HomeSectionIF.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/interfaces/mhr-registration-interfaces/HomeSectionIF.ts:2]: Interesting, we could definitely initialize store with `0`!

I'll reach out to UX if that is also a default they would expect in the UI, my assumption is that they would prefer to prompt a user to input a value using the usual validation prompts.

Thank you for the heads up!

File: ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts]: Updated 👍

File: ppr-ui/src/store/state/state-model.ts (hunk 3 above)
Comment (cameron-eyds) [reply ppr-ui/src/store/state/state-model.ts]: Updated 👍

**Comment by cameron-eyds** [comment]
@dimak1 @garyfan1 
Going to merge, as this has been open for 20hrs and was committed to your benefit to avoid conflicts. 
If you have any comments, don't hesitate to put them here, thanks.

**Pull request merged**